.gitignore

```diff
@@ -29,6 +29,7 @@ data/**/*.html
 !data/raw_data/sample-mint/*
 !data/parsed_data/sample-mint/*
 !data/processed_data/sample-mint/*
+!data/query_response_data/sample-mint/*
```

Comment:
Kindly remove this.

Reply:
@MadaraUchiha-314 If I remove this then the "data/query_response_data/sample-mint/" folder structure is not created in the repository because of which write_query_results function in utils.py fails with an error of 'File not found'.
And GitHub doesn't allow empty folders in a commit, hence I have added the files but removed any data.

Reply:
You need to check whether the folder exists before you write something to it.

Look at this: https://github.com/intuit/fawkes/blob/master/fawkes/parse/parse.py#L204